assets/card.css

```diff
@@ -0,0 +1,9 @@
+.fixedCard {
+  @apply flex flex-wrap rounded-10;
```

Comment:
According to abstract the radius is 12px

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/3371760/89985395-48e0c680-dc7b-11ea-9053-d61b561f02e6.png">
https://app.abstract.com/projects/0935fec8-eab2-420e-b519-c7c185935030/branches/f98ccf2a-d63b-42fb-9f13-64de4a7272e5/commits/a45d120cfe8c08ba5f6ae18b59ac4b454d611b8d/files/27b5d2a6-1049-4b8c-91d2-026563d2d288/layers/9DED07A7-FCF3-49BD-AB83-0AD868F2112D?collectionId=19c9b4c0-fc5b-4260-9413-c98d4c7317d1&collectionLayerId=43f3f259-4973-4260-a18e-a03ccafc16f6&mode=build&selected=3076742694-09BC476B-E998-4AC2-A867-F984AE49584A

Reply:
The cards have `rounded-10`.
So, for consistency we leave it with 10px for all the cards.

Reply:
And I have checked with Enrique and he agrees with 10px for all the cards.